exchange/auction_response.go

```diff
@@ -0,0 +1,20 @@
+package exchange
+
+import (
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// AuctionResponse contains OpenRTB Bid Response object and its extension (un-marshalled) object
+type AuctionResponse struct {
+	*openrtb2.BidResponse                             // BidResponse defines the contract for openrtb bidresponse
+	ExtBidResponse        *openrtb_ext.ExtBidResponse // ExtBidResponse defines the contract for bidresponse.ext
+}
```

Comment:
@SyntaxNode : embedded `openrtb2.BidResponse`, because, I was looking at  `AuctionResponse` is extended version of `openrtb2.BidResponse`. So, it would be easy for consumer to access fields of  `openrtb2.BidResponse` directly, without doing `auctionResopnse.Bid` or similar